src/ucs/sys/math.h

```diff
@@ -85,6 +85,9 @@ BEGIN_C_DECLS
 #define ucs_div_round_up(_n, _d) \
     (((_n) + (_d) - 1) / (_d))
 
+#define ucs_signum(_n) \
```

Comment:
what is the purpose of this macro?

Reply:
move to gtest - it is needed to compare expected and actual results